doc/usage/configuration.rst

```diff
@@ -660,10 +660,11 @@ documentation on :ref:`intl` for details.
    generated by Sphinx will be in that language.  Also, Sphinx will try to
    substitute individual paragraphs from your documents with the translation
    sets obtained from :confval:`locale_dirs`.  Sphinx will search
-   language-specific figures named by `figure_language_filename` and substitute
-   them for original figures.  In the LaTeX builder, a suitable language will
-   be selected as an option for the *Babel* package.  Default is ``None``,
-   which means that no translation will be done.
+   language-specific figures named by `figure.language.filename` 
+   (e.g. the German version of `myfigure.png` will be `myfigure.de.png`) 
```

Comment:
Looks better. But ``` `` ``` (double back-ticks) are better for a literal string in reStructuredText. I think the original text `figure_language_filename` is also not good. Could you use it instead of a single one?

Reply:
Changed it for the filename example. Should I also change it for the figure.language.filename?

Reply:
I should be fixed also. But I'll do it on another PR.